SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs

```diff
@@ -34,52 +34,140 @@ public class SpeechSandboxStreaming : MonoBehaviour
     public AudioClip sorryClip;
     public List<AudioClip> helpClips;
 
+    [SerializeField]
+    private fsSerializer _serializer = new fsSerializer();
 
+    #region PLEASE SET THESE VARIABLES IN THE INSPECTOR
+    [Header("Speech To Text")]
+    [Tooltip("The service URL (optional). This defaults to \"https://stream.watsonplatform.net/speech-to-text/api\"")]
+    [SerializeField]
+    private string speechToTextServiceURL = "";
+    [Header("CF Authentication")]
+    [Tooltip("The authentication username.")]
+    [SerializeField]
+    private string speechToTextUsername = "";
+    [Tooltip("The authentication password.")]
+    [SerializeField]
+    private string speechToTextPassword;
+    [Header("IAM Authentication")]
+    [Tooltip("The IAM apikey.")]
+    [SerializeField]
+    private string speechToTextIAMApiKey;
+    [Tooltip("The IAM url used to authenticate the apikey (optional). This defaults to \"https://iam.bluemix.net/identity/token\".")]
+    [SerializeField]
+    private string speechToTextIAMURl;
```

Comment:
This should be
```
private string speechToTextIamUrl;
```

Reply:
*sigh* You are right. And it renders funny in the editor.